main.go

```diff
@@ -101,6 +104,7 @@ func argsParse(argv []string) (finder.Args, error) {
 		args.SRIOVConfigFile = path
 	}
 	args.SysfsRoot = arguments["--sysfs"].(string)
+	args.ContainerRuntime = arguments["--container-runtime"].(string)
```

Comment:
nice, but let's add an explicit validation here, for better UX and more robust code. So the user can know asap which runtimes are supported and how we identify them.

Reply:
I have added another commit for input validation